src/modules/authentication/api/authentication.controller.ts

```diff
@@ -97,9 +99,10 @@ export class AuthenticationController {
     @ApiUnauthorizedResponse({ description: 'User is not logged in.' })
     @ApiOkResponse({ description: 'Returns info about the logged in user.', type: UserinfoResponse })
     public async info(@Permissions() permissions: PersonPermissions): Promise<UserinfoResponse> {
-        const roleIds: string[] = await permissions.getRoleIds();
+        const roleIds: RolleID[] = (await permissions.getRoleIds()).map((item: RolleID) => item);
         this.logger.info('Roles: ' + roleIds.toString());
         this.logger.info('User: ' + JSON.stringify(permissions.personFields));
-        return new UserinfoResponse(permissions);
+        const rolleFields: PersonenkontextRolleFields[] = await permissions.getPersonenkontextewithRoles();
+        return new UserinfoResponse(permissions, rolleFields);
```

Comment:
Either map the `PersonenkontextRolleFields[]` to `PersonenkontextRolleFieldsResponse[]` here or in the UserinfoResponse. Otherwise the response wont be an instance of the `PersonenkontextRolleFieldsResponse`-class